.github/issue_template.md

```diff
@@ -0,0 +1,8 @@
+<!---
+Thank you for reporting a problem and suggesting improvements. Please provide the below information to make sure we can effectively deal with the issue reported. For the most precise version information, see the first line of console output or run with --version.
+--->
+
+CBMC version:
+Operating system:
```

Comment:
Compiler name and version (perhaps including advice how to determine what compiler cbmc / goto-cc is using for preprocessing)

Reply:
Is this a common source of problems? I'd like to keep it as simple as possible, but I am of course aware that this is very useful information at times.

Reply:
If there's a simple cbmc command-line to print that information then we might as well ask; if it's more involved then of course we can ask later if it seems relevant.

Reply:
I'm not aware of such an option so I'll merge this with the current state. Happy to revisit later.